db/schema.rb

```diff
@@ -280,6 +280,7 @@
     t.bigint "event_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "theme", default: "default"
```

Comment:
Can we make this null: false also?

Reply:
I guess I was thinking that the default would be sufficient but I suppose someone could clear it from the console. I will make that addition.